includes/shortcodes/class.llms.shortcodes.blocks.php

```diff
@@ -163,34 +165,24 @@ public function render_block( array $attributes, string $content, WP_Block $bloc
 			return '';
 		}
 
-		$name = str_replace(
-			array( 'llms/', '-' ),
-			array( '', '_' ),
-			$block->name
-		);
-
-		$atts = '';
-
-		foreach ( $attributes as $key => $value ) {
-			if ( empty( $value ) || 'text' === $key ) {
-				continue;
-			}
+		$name   = str_replace( 'llms/', '', $block->name );
+		$class  = $this->shortcodes[ $name ][0];
+		$method = $this->shortcodes[ $name ][1];
 
-			if ( strpos( $key, 'llms_' ) !== false ) {
-				continue;
-			}
-
-			$atts .= " $key=$value";
+		if ( method_exists( $class, 'instance' ) ) {
+			$class = $class::instance();
```

Comment:
this is more `$object` than `$class`.
or `$renderer_instance`? even better probably